src/views/AccountDetails/AccountDetails.js

```diff
@@ -670,6 +654,48 @@ class AccountDetails extends React.Component {
               />
             </Col>
           </Row>
+          <Row>
+            <Col xs={1.5} sm={1.5} md={1.5} lg={1.5}>
```

Comment:
Since `xs` is the default and things scale up for there, the other `<Col>` styles (`sm`, `md`, `lg`) can be omitted when they match.

Reply:
Apply the recommendations or fixes in my new commit